### PR TITLE
[16.0][FIX] website_sale_product_pack: Migration to 16.0

### PR DESCRIPTION
**Pull request by lef-adhoc**
This is a continuation of the pr #135
[See more info here](https://github.com/odoo/odoo/commit/b121f8e1d3ca5b98e71e25710427e6210afe9ef6)

### CODE REVIEW

**Review by ioans73 — APPROVED** [review]
LGTM 👍🏻

**Comment by ioans73** [comment]
@OCA/product-maintainers could you review this? (is a continuation of the pr https://github.com/OCA/product-pack/pull/135)

cc @alvaro-domatix

**Review by NICO-SOLUTIONS — COMMENTED** [review]
LGTM

**Comment by ioans73** [comment]
> LGTM

@NICO-SOLUTIONS can you approve?

**Review by NICO-SOLUTIONS — APPROVED** [review]

**Comment by ioans73** [comment]
@pedrobaeza Could we merge this?

File: website_sale_product_pack/__manifest__.py [hunk 1]
```diff
@@ -4,10 +4,15 @@
     "name": "Website Sale Product Pack",
     "category": "E-Commerce",
     "summary": "Compatibility module of product pack with e-commerce",
-    "version": "13.0.1.0.2",
+    "version": "16.0.1.0.3",
```
Comment (pedrobaeza) [inline website_sale_product_pack/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

**Review by pedrobaeza — COMMENTED** [review]
@chienandalu is it OK for you?

**Comment by ioans73** [comment]
> @chienandalu is it OK for you?

@chienandalu could you review this please?

File: website_sale_product_pack/__manifest__.py (hunk 1 above)
Comment (ioans73) [reply website_sale_product_pack/__manifest__.py]: ping @lef-adhoc

Comment (lef-adhoc) [reply website_sale_product_pack/__manifest__.py]: @ioans73 I have already made the change, is it ok?

Comment (ioans73) [reply website_sale_product_pack/__manifest__.py]: @pedrobaeza If you could review again...

**Pull request merged**